.github/workflows/build_deploy_wheels.yml

```
@@ -73,15 +87,26 @@ jobs:
           python setup.py sdist bdist_wheel
         shell: powershell
 
-      - name: Publish wheels to PyPI
-        if: startsWith(github.ref, 'refs/tags')
+
+      - name: Build Solcore MacOS
+        if: matrix.os != 'windows-latest'
         env:
-          TWINE_USERNAME: '__token__'
-          TWINE_PASSWORD: ${{ secrets.PYPI_TOKEN }}
+          FC: gfortran
+          CC: gcc
+          SOLCORE_WITH_PDD: 1
         run: |
-          twine upload dist/*-win_amd64*.whl --skip-existing
+          export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/usr/local/lib/
+          python setup.py sdist bdist_wheel
+
+#      - name: Publish wheels to PyPI
+#          if: startsWith(github.ref, 'refs/tags')
+#          env:
+#            TWINE_USERNAME: '__token__'
+#            TWINE_PASSWORD: ${{ secrets.PYPI_TOKEN }}
+#          run: |
+#            twine upload dist/*.whl --skip-existing
 
       - uses: actions/upload-artifact@v3
         with:
-          name: windows_wheels
-          path: dist/
+          name: OS wheels
```

Maybe make it OS dependent, as this is for both Mac and Windows.
```suggestion
          name: ${{ matrix.os }}_wheels
```